### ISSUE TITLE
Word equations in col_keys and footnote

### ISSUE THREAD

**Issue by bwiernik** [davidgohel/flextable#68]
I am trying to use `flextable::regulartable` to output a table that includes LaTeX math syntax in the col_keys and in the footnote (specifically, I am trying to use RMarkdown to render a .docx document). If I use `knitr::kable`, the math syntax is converted into Word equations when output to .docx. However, if I use `flextable::regulartable`, the syntax is not converted and rendered as-is. Is there any way to get flextable to render Word equations, and if not, is there any possibility this functionality might be added to flextable/officer?

$\mathbfit{SD_{r}}$
$\mathbfit{SD_{res}}$
$\mathbf{\overline{\rho}}$
$\mathbfit{SD_{r_{c}}}$
$\mathbfit{SD_{\mathbf{\rho}}}$

**Comment by davidgohel** [comment]
Hi 

Sorry for the delay. I don't have a solution for that now but I'd like to.

**Comment by davidgohel** [comment]
I am closing that issue as I still did not have time for it. I may reopen later

**Comment by rdisalv2** [comment]
@davidgohel do you think it'd be possible to work with body_add_xml in officer to add math mode equations? if you think so i'll take a look and see if i can get some rudimentary equations working. thanks!

**Comment by davidgohel** [comment]
@rdisalv2 Hi. What is hard there is to be able to provide the format expected by Word (https://docs.microsoft.com/fr-fr/dotnet/api/documentformat.openxml.math?view=openxml-2.8.1). This will be extremely painful... 

At ArData, we thought about it and gave away because we lack the time to do it, I think it's a huge work (but I may be wrong!). Plus, people will want to write latex, so it not only means to be able to embed the equations but also transform them from latex to openxml format.

However, if you find a solution, I will be happy merge the PR :)

**Comment by rdisalv2** [comment]
@davidgohel thanks for your response. but, at a high level body_add_xml could do the trick? i'm thinking of starting with some specific things we need regularly for academic papers, like some greek for table labels, instead of implementing the whole spec. the xml i'm seeking didn't look too complicated when i looked at it from word. agreed about the latex to openxml, that would be a bear.

**Comment by davidgohel** [comment]
OK, sorry for my misunderstanding 

Adding greek letters, symbols, etc is already possible. There is a simple example here: https://davidgohel.github.io/flextable/articles/display.html#sugar-functions-for-complex-formatting

<img width="138" alt="Capture d’écran 2019-08-20 à 18 03 11" src="https://user-images.githubusercontent.com/4331618/63363912-d0e80500-c374-11e9-9e25-744da99121e7.png">


Is that what you are after? 

If not, could you post a screen shot so that I can check there is no solution?

PS: flextable is not using `body_add_xml` (too slow). Instead, it create the xml.

**Comment by rdisalv2** [comment]
this is great, almost what i need.

question: can i store the  as_paragraph(as_b("µ"), as_sup("blah"))  object in a cell of a data.frame then have flextable convert it later somehow? i couldn't figure out the notation for that.

(btw, happy to post this as a separate issue or SO question since we've gotten away from math mode...)

**Comment by davidgohel** [comment]
I don't know. Probably with rlang (used in flextable also)...

Yes, please open a separate issue with what you want to achieve and I will try to help.

(and sorry, I will not be able to answer during the following next days)

**Comment by davidgohel** [comment]
There is support for 'MathJax' equations since `flextable 0.6.5`. For now, only by using `as_equation`:


* The Word and PowerPoint results are filled with a real Word/PPT equation
* The HTML version is an SVG image
* The PDF version is a latex equation

```
eqs <- c(
  "(ax^2 + bx + c = 0)",
  "a \\ne 0",
  "x = {-b \\pm \\sqrt{b^2-4ac} \\over 2a}")
df <- data.frame(formula = eqs)
df


ft <- flextable(df)
ft <- compose(
  x = ft, j = "formula",
  value = as_paragraph(as_equation(formula, width = 2, height = .5)))
ft <- align(ft, align = "center", part = "all")
ft <- width(ft, width = 2)
ft
```

![flextable-012](https://user-images.githubusercontent.com/4331618/114839838-87182200-9dd6-11eb-9a43-c06dc9093998.png)

**Comment by daniel1noble** [comment]
Hi,

I'd love to use the `as_equation()`  to render equations within  my tables, but I am having some difficulty getting this to work. I've followed the instructions [here:](https://davidgohel.github.io/flextable/reference/as_equation.html): ie.e., "To use this function, package 'equatags' is required; also equatags::mathjax_install() must be executed only once to install necessary dependencies" I've installed the `equatags` package. However, even running the example code chunk it fails with the error: Error in as_equation(formula, width = 2, height = 0.5) :   could not find function "as_equation" 

Is anyone else having difficulty? Here is my sessionInfo

`R version 4.0.5 (2021-03-31)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Catalina 10.15.7

Matrix products: default
BLAS:   /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libBLAS.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/4.0/Resources/lib/libRlapack.dylib

locale:
[1] en_AU.UTF-8/en_AU.UTF-8/en_AU.UTF-8/C/en_AU.UTF-8/en_AU.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] equatags_0.1.0  flextable_0.6.4

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.6        bookdown_0.21     packrat_0.6.0     digest_0.6.27     rappdirs_0.3.3    R6_2.5.0         
 [7] evaluate_0.14     zip_2.1.1         rlang_0.4.10      gdtools_0.2.3     uuid_0.1-4        data.table_1.14.0
[13] xml2_1.3.2        rmarkdown_2.7     locatexec_0.1.0   xslt_1.4.2        tools_4.0.5       officer_0.3.18   
[19] rsconnect_0.8.16  xfun_0.22         yaml_2.2.1        compiler_4.0.5    systemfonts_1.0.1 base64enc_0.1-3  
[25] htmltools_0.5.1.1 knitr_1.31 `

**Comment by davidgohel** [comment]
@daniel1noble There is support for 'MathJax' equations since `flextable 0.6.5`, you need to update your package.

**Comment by daniel1noble** [comment]
@davidgohel Of course. That was a dumb question! Thanks for catching that. I did a fresh install of flextable thinking I had the most updated version, but clearly I did not! All is working now.
